devkit/e2e/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "rpch-e2e",
+  "version": "0.0.0",
+  "main": "./build/index.js",
+  "types": "./build/index.d.ts",
+  "license": "MIT",
+  "scripts": {
+    "build": "exit 0",
+    "dev": "yarn build --watch",
+    "format": "prettier --check \"src/**/*.{ts,md}\"",
+    "lint": "TIMING=1 eslint \"src/**/*.ts*\"",
+    "test": "exit 0",
```

Comment:
can we make the test function here run the sh script & all the tests?

Reply:
Good idea! Done ✅